src/quest_manager/models.py

```diff
@@ -127,8 +127,7 @@ class XPItem(models.Model):
     )
     archived = models.BooleanField(
         default=False,
-        help_text='Setting this will prevent it from appearing in admin quest lists.  '
-        'To un-archive a quest, you will need to access it through Site Administration.'
+        help_text="Move this quest into the Archived tab."
```

Comment:
We'll need to update this help_text with more info once we decide what else happens when you archive a quest (for example removing it as a prerequisite, deleting any submissions, etc)